### ISSUE TITLE
working script for loop is failing on this editor.

### ISSUE THREAD

**Issue by swtimd79sv** [koalaman/shellcheck#2882]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC1073, SC1072
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that has actually worked, but the shell check says its got error

```sh
for account in $(aws organizations list-accounts --query 'Accounts[].Id' --output text); do


```

#### Here's what shellcheck currently says:
```
Line 1	SC1073: Couldn't parse this for loop. Fix to allow more checks.
Line 3	SC1072: Expected a command. Fix any mentioned problems and try again.
```
#### Here's what I wanted or expected to see:

TODO: Describe expected/desired output

**Comment by brother** [comment]
That example is not complete. 

The for loop is broken as it has no body and no done statement. Adding those will make shellcheck parse it fine. But you need to indicate what shell to act as.

Consider the following complete (and shellcheck clean) example.

```shell
#!/bin/bash

for account in $(aws organizations list-accounts --query 'Accounts[].Id' --output text); do
echo "$account"
done
```
